frontend/src/pages/org/workflow-detail.ts

```diff
@@ -119,6 +124,7 @@ export class WorkflowDetail extends LiteElement {
   private readonly tabLabels: Record<Tab, string> = {
     crawls: msg("Crawls"),
     watch: msg("Watch Crawl"),
+    logs: msg("Logs"),
```

Comment:
This is "Error Logs" in the crawl - might be good to be consistent for now until we display full logs?